### PR TITLE
Replace MUI Link component for the Sign In page due to a conflict with router-DOM Link component.

### PR DESCRIPTION
**Pull request by UserOlena**
Due to a conflict, the MUI Link component had to be removed and replaced with an <a> tag. This change was necessary because the react-router-DOM Link component is required for routing pages within the application.

Removed unused code for regEx validation.

### CODE REVIEW

File: client/src/pages/SignUp.js [hunk 1]
```diff
@@ -27,13 +28,15 @@ function Copyright(props) {
...
+        target='_blank'
         href='https://github.com/UserOlena/Learnify/blob/main/LICENSE'
+        style={{
+          color: 'rgba(0, 0, 0, 0.6)',
+        }}
       >
         Learnify
-      </Link>
-      {' '}
+      </a>{' '}
```
Comment (sendusyourbones) [inline client/src/pages/SignUp.js]: Should `{' '}` be on a new line?

File: client/src/pages/SignUp.js [hunk 2]
```diff
@@ -91,7 +94,7 @@ export function SignUp() {
       return;
     }
 
-    // if values of both password and confirmPassword fields !== 0, ensure they
+    // if values lenfth of both password and confirmPassword fields !== 0, ensure they
```
Comment (sendusyourbones) [inline client/src/pages/SignUp.js]: This needs to be fixed again, sorry :)

Comment (UserOlena) [reply client/src/pages/SignUp.js]: I'm sorry about that :)

**Review by sendusyourbones — APPROVED** [review]
Looks good!

**Pull request merged**